### DISCUSSION TITLE
writing of related_label

### DISCUSSION THREAD

**Discussion by jphconstantin** [mmistakes/minimal-mistakes#2663] [Q&A] [answered]
Small details: if the string **related_label** in ui_text.yml is written in lower case, single.html display it in uppercase.
In fact the user has no choice.
Desired by the author ?

**Comment by mmistakes** [comment]
Feel free to submit a pull request to make them consistent.

**Comment by iBug** [comment]
It's controlled by CSS. Add the following CSS as your custom to override it and restore the default behavior:

```scss
.page__share-title {
  text-transform: initial;
}
```

**Comment by jphconstantin** [comment]
Yes but why don't you implement it in the theme ?

**Comment by mmistakes** [comment]
Maybe I'm misunderstanding what the issue is. The `related_label` is 100% controllable by you the user. There is a [default string](https://github.com/mmistakes/minimal-mistakes/blob/master/_layouts/single.html#L68) (`You May Also Enjoy`) that is applied if you don't have a [ui-text.yml](https://github.com/mmistakes/minimal-mistakes/blob/master/_data/ui-text.yml#L29) data file. But if you have that data file in your repo, then that whatever text you put in there will be used instead.

So not exactly sure what you're getting at with the theme doing it. At first I thought you meant there was an inconsistency with title casing between the default string and what is in the *sample* data file (e.g. `You may also enjoy`). Which if that is the case feel free to submit a pull request changing it so they're both the same titling.

Regardless, doesn't really matter either way as that data file is something the user copies to their local repo and *should* tailor the text to meet their needs. If you don't like the initial caps text... change it. If you don't like the way things are described... change it. This was my intention with providing a file with all the text strings so they could be changed by the user easily.

**Comment by mmistakes** [comment]
And what @iBug was getting at, the theme's default styling for those types of headings is to uppercase them. It's purely decorative and part of the theme's design.

If you want to change the design, then you can use the CSS above to override that. This theme does have some opinions which is part of the reason it is used so widely. It is in no way meant to be a site builder where you can go in and have options to change every little things via simple configs.

**Comment by jphconstantin** [comment]
The text is in lower case in ui file but the display is un upper. No reason
to change it because always displayed in upper !

Le lun. 24 août 2020 19:16, Michael Rose <notifications@github.com> a
écrit :

**Comment by mmistakes** [comment, marked as answer]
@jphconstantin That is by design. The text isn't uppercase, it's just styled to be that way.

![image](https://user-images.githubusercontent.com/1376749/91076172-1769e280-e60d-11ea-906a-09670429870a.png)

**Comment by jphconstantin** [comment]
Ok. You can close the subject.

Le lun. 24 août 2020 19:24, Michael Rose <notifications@github.com> a
écrit :
